docs/cudf/source/user_guide/performance_comparisons/performance-comparisons.ipynb

```diff
@@ -8,13 +8,16 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "This notebook compares the performance of `cuDF` and `pandas`. The comparisons performed are on identical data sizes. This notebook primarily showcases the factor\n",
     "of speedups users can have when the similar `pandas` APIs are run on GPUs using `cudf`.\n",
     "\n",
-    "The hardware details used to run these performance comparisons are at the end of this page."
+    "The hardware details used to run these performance comparisons are at the end of this page.\n",
+    "\n",
+    "**Note**: This notebook is written to measure performance on modern NVIDIA hardware, for older NVIDIA hardware with lower GPU memory please consider lowering the `num_rows` values below by a factor of 2. Results may vary by data sizes, CPU & GPU used."
```

Comment:
I'm not sure that we should differentiate old/new here. We should focus on high-memory vs. low-memory. There are a range of relatively new consumer GPUs that would probably have too little memory for the largest data sizes here.
```suggestion
    "**Note**: This notebook is written to measure performance on NVIDIA GPUs with large memory. If running on hardware with lower memory, please consider lowering the `num_rows` values. Performance results may vary by data size, as well as the CPU and GPU used."
```